packages/backend/server/src/plugins/indexer/__tests__/init-ms.sh

```
@@ -0,0 +1,1365 @@
+
+## Create a index table with a mapping
+
+# https://manual.manticoresearch.com/Creating_a_table/Local_tables/Real-time_table
+
+curl -v -X DELETE "http://localhost:9308/doc" | json
+curl -v -X POST http://127.0.0.1:9308/cli -H "content-type: application/plain" -H "accept: application/json" --data-binary @./packages/backend/server/src/plugins/indexer/tables/doc.sql
+curl -v -X POST http://127.0.0.1:9308/cli --data-binary @./packages/backend/server/src/plugins/indexer/tables/doc.sql
+curl -v -X POST "http://localhost:9308/cli" -d "DESC doc"
+
++--------------------+------------+----------------+
+| Field              | Type       | Properties     |
```

Shellcheck (0.10.0)

[error] Parsing stopped here. Mismatched keywords or invalid parentheses?

(SC1070)